src/scopes/activityReport/index.test.js

```diff
@@ -355,6 +355,37 @@ describe('filtersToScopes', () => {
         expect(found.map((f) => f.id))
           .toEqual(expect.arrayContaining([reportExcluded.id, globallyExcludedReport.id]));
       });
+
+      it('grant number with matches', async () => {
```

Comment:
For consistency, we might want to consider moving the grant number tests into their own `describe ` block.

Reply:
I put it in the describe block for 'Grants' to prevent having duplicate test setup code run.

The same thought crossed my mind.